polymer/src/abe-polymer2-app.html

```diff
@@ -0,0 +1,227 @@
+<link rel="import" href="../node_modules/@npm-polymer/polymer/polymer-element.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-route/app-location.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-route/app-route.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-media-query/iron-media-query.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-ajax/iron-ajax.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-pages/iron-pages.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-selector/iron-selector.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-layout/app-scroll-effects/effects/waterfall.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-layout/helpers/helpers.html">
+<link rel="import" href="sample-home.html">
+<link rel="import" href="sample-list.html">
+
+<dom-module id="abe-polymer2-app">
+  <template>
+  <style>
+
+    :host {
+      display: block;
+      position: relative;
+      padding-top: 130px;
+      padding-bottom: 64px;
+      min-height: 100vh;
+      --app-primary-color: #202020;
+      --app-secondary-color: #757575;
+      --app-accent-color: #172C50;
+      --paper-button-ink-color: var(--app-accent-color);
+      --paper-icon-button-ink-color: var(--app-accent-color);
+      --paper-spinner-color: var(--app-accent-color);
+      -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+      color: var(--app-primary-color);
+    }
+
+    paper-icon-button {
+      color: var(--app-primary-color);
+    }
+
+    :host(:not([page=detail])) .back-btn {
+      display: none;
+    }
+
+    [hidden] {
+      display: none !important;
+    }
+
+    iron-pages {
+      max-width: 1440px;
+      margin: 0 auto;
+    }
+
+    /* small screen */
+    @media (max-width: 767px) {
+      :host {
+        padding-top: 64px;
+      }
+    }
+
+  </style>
+    <app-location route="{{route}}"></app-location>
+    <app-route
+        route="{{route}}"
+        pattern="/polymer/:page"
+        data="{{routeData}}"
+        tail="{{subroute}}"></app-route>
+
+  <iron-ajax auto url="https://ampbyexample.com/sitemap.json" handle-as="json"
+    on-response="handleResponse"></iron-ajax>
```

Comment:
This can now also use `sample-category-data`

Reply:
I think this can still be fixed. Maybe, even that the sub-pages use `sample-category-data` instead of the app element passing the data down.

Reply:
Done